articles/connections/social/linkedin.md

```diff
@@ -8,67 +8,67 @@ description: This page shows you how to connect your Auth0 app to LinkedIn. You
 toc: true
 ---
 
-# Connect your app to LinkedIn
+# Connect Your App to LinkedIn
 
 To connect your Auth0 app to LinkedIn, you will need to generate a **Client ID** and **Client Secret** in a LinkedIn app, copy these keys into your Auth0 settings, and enable the connection.
 
-## 1. Log into the developer portal
+## 1. Log in to the Developer Portal
 
-Login to the [LinkedIn Developer portal](http://developer.linkedin.com/) and click **My Apps**:
+Log in to the [LinkedIn Developer portal](http://developer.linkedin.com/), and click **My Apps**:
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-1.png)
 
-## 2. Create your app
+## 2. Create Your App
 
 Click the **Create Application** button:
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-2.png)
 
-## 3. Complete information about your app
+## 3. Enter Details About Your App
 
-For the **Website URL** field enter the following URL:
+For the **Website URL** field, enter the following URL:
 
 `https://${account.namespace}`
 
-Then complete all the required feilds on the form. 
+Then complete all the required fields on the form. 
 
 Click **Submit**.
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-3.png)
 
-## 4. Enter Redirect URL
+## 4. Enter the Redirect URL
 
-Enter the following URL in the **Authorized Redirect URLs** field and click **Add**:
+Enter the following URL in the **Authorized Redirect URLs** field, and click **Add**:
 
 `https://${account.namespace}/login/callback`
 
-Next click **Update**.
+Next, click **Update**.
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-4.png)
 
-## 5. Get your **Client ID** and **Client Secret**
+## 5. Get Your **Client ID** and **Client Secret**
```

Comment:
No bold in subheads, the `## 5. Get Your **Client ID** and **Client Secret**` should be `## 5. Get your Client ID and Client Secret`